file/helper_windows.go

```diff
@@ -23,9 +23,13 @@ import (
 )
 
 // SafeFileRotate safely rotates an existing file under path and replaces it with the tempfile
-func SafeFileRotate(path, tempfile string) error {
+func SafeFileRotate(path, tempfile string, opts ...RotateOpt) error {
+	options := rotateOpts{}
```

Comment:
I think default we should have the `renameRetryDuration` and `renameRetryInterval` set to the defaults on Windows. At the moment you have to use the options to get the behavior, I think having it the default is more preferred on Windows.

Reply:
Addressed in 04a4d5e.